### DISCUSSION TITLE
[Feature] Subtasks

### DISCUSSION THREAD

**Discussion by srkunze** [Kozea/Radicale#1373] [Ideas]
Hi everyone,

first I want to say “thank you for this extremely lightweight server”. I love it. It works perfectly on my Raspberry Pi 4. 😊

Second, I‘m curious if there are any plans to integrate subtasks. iOS supports them and I would like to use them.

Best
Sven

**Comment by luckydonald** [comment]
I'm curious as well. Are they part of the official CalDAV standard?

**Comment by smithfred** [comment]
Apple Calendar, NextCloud and others (e.g. Errands on GNOME), use the RELATED-TO field for this purpose, see for example https://github.com/dmfs/opentasks/issues/341.

Can the related issue for this be unlocked on this basis?

**Reply by LSeelig** [reply]
I am not an advanced user, but I downloaded the lists from the webui as ics, and in the text editor, I saw the "related-to" field was intact in radicale?

So what is the problem? It seems like it is being synced.

**Reply by pbiering** [reply]
_radicale_ is not dealing with any content in ICS or VCF, either it's parsed proper and accepted by _vobject_ or rejected. So I would assume this issue can be closed.

**Comment by smithfred** [comment]
iCalendar RFC specifically defines the field as having default "referred-to object is parent" semantics too: https://datatracker.ietf.org/doc/html/rfc5545#section-3.8.4.5
